project/AppDependencies.scala

```diff
@@ -1,27 +1,26 @@
 import sbt._
 import play.core.PlayVersion
 import play.sbt.PlayImport._
-import uk.gov.hmrc._
 
 object AppDependencies {
 
   val silencerVersion = "1.7.1"
 
   val compile: Seq[ModuleID] = Seq(
     ws,
-    "uk.gov.hmrc" %% "bootstrap-backend-play-26" % "3.2.0",
-    "uk.gov.hmrc" %% "domain" % "5.10.0-play-26",
-    "uk.gov.hmrc" %% "http-caching-client" % "9.2.0-play-26",
-    "uk.gov.hmrc" %% "tabular-data-validator" % "1.3.0",
+    "uk.gov.hmrc" %% "bootstrap-backend-play-27" % "5.3.0",
+    "uk.gov.hmrc" %% "domain" % "5.11.0-play-27",
+    "uk.gov.hmrc" %% "http-caching-client" % "9.5.0-play-27",
+    "uk.gov.hmrc" %% "tabular-data-validator" % "1.4.0",
     "org.scala-lang.modules" %% "scala-xml" % "1.0.6",
-    "org.scala-lang.modules" %% "scala-parser-combinators" % "1.0.6",
-    "com.typesafe.play" %% "play-json-joda" % "2.6.10",
-    "uk.gov.hmrc" %% "auth-client" % "3.2.0-play-26",
+    "org.scala-lang.modules" %% "scala-parser-combinators" % "1.0.7",
+    "com.typesafe.play" %% "play-json-joda" % "2.6.14",
+    "uk.gov.hmrc" %% "auth-client" % "3.3.0-play-27",
```

Comment:
auth-client is brought in by bootstrap so it can be removed here I believe

Reply:
Nice, will remove this.